Typescript error with IntegratedSummary

<!--
    If you have any questions regarding your scenario or components behavior, please provide a detailed explanation.
    If you own an active DevExtreme license, feel free to contact us in the Support Center (https://www.devexpress.com/ask) to receive an answer shortly.
-->

<!--
    Checked checkbox should look like this: [x]
-->
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### I'm using ...
<!--
    Checked checkbox should look like this: [x]
-->
- [x] React Grid
- [ ] React Chart
- [ ] Vue Grid

### Current Behaviour
I got TypeScript error like this

`error TS2322: Type '{}' is not assignable to type 'IntegratedSummaryProps'.
  Property 'calculator' is missing in type '{}'.`

when trying to use `<IntegratedSummary />`

### Expected Behaviour



### Steps to Reproduce (for Bugs)
<!--
    Provide detailed steps to reproduce the bug. If possible, create a demo illustrating the problem.
-->



### Environment
<!---
    Include as many relevant details about the environment with which you experienced the bug. Leave "none" if you don't have the package in your package.json file.
-->

- typescript: 3.0.1
- devextreme-reactive: 1.6.1
- react: 16.4.2
- vue: xxx
- browser: xxx
- bootstrap: none
- react-bootstrap: none
- material-ui: 1.5.0

--- a/packages/dx-react-grid/docs/reference/custom-summary.md
+++ b/packages/dx-react-grid/docs/reference/custom-summary.md
@@ -20,9 +20,9 @@ import { CustomSummary } from '@devexpress/dx-react-grid';
 
 Name | Type | Default | Description
 -----|------|---------|------------
-totalValues | Array&lt;any&gt; | | Total summary values.
-groupValues | { [key: [GroupKey](grouping-state.md#groupkey)]: Array&lt;any&gt; } | | Group summary values.
-treeValues | { [key: number &#124; string]: Array&lt;any&gt; } | | Tree summary values.
+totalValues? | Array&lt;any&gt; | | Total summary values.
+groupValues? | { [key: [GroupKey](grouping-state.md#groupkey)]: Array&lt;any&gt; } | | Group summary values.
+treeValues? | { [key: number &#124; string]: Array&lt;any&gt; } | | Tree summary values.
 
 ## Plugin Developer Reference
 

--- a/packages/dx-react-grid/docs/reference/integrated-summary.md
+++ b/packages/dx-react-grid/docs/reference/integrated-summary.md
@@ -20,7 +20,7 @@ import { IntegratedSummary } from '@devexpress/dx-react-grid';
 
 Name | Type | Default | Description
 -----|------|---------|------------
-calculator | (type: [SummaryType](summary-state.md#summarytype), rows: Array&lt;any&gt;, getValue: (row: any) => any) => any | | A summary calculator.
+calculator? | (type: [SummaryType](summary-state.md#summarytype), rows: Array&lt;any&gt;, getValue: (row: any) => any) => any | | A summary calculator.
 
 ## Static Fields
 

